### PR TITLE
Add Right to Left capability to Telegram for languages like arabic, urdu and persian

### PR DESCRIPTION
**Pull request by AMP999**
this capability can support languages like:
1.Arabic
2.Persian/Farsi
3.Azeri
4.Aramaic
5.Dhivehi/Maldivian
6.Hebrew
7.Kurdish (Sorani)
8.Urdu

i'm working on this project. till now dialogs are almost fixed

### CODE REVIEW

**Comment by AMP999** [comment]
> If the PR is still WIP, you can use the GitHub's draft feature

A complete fix of rtl will be a huge PR. so I decided to PR it when a hunk is ready.
fix of rtl won't interfere with existing experience as some parts of the telegram are already rtl compatible.

**Comment by ilya-fedin** [comment]
So you want to say your PR is ready for review?

**Comment by AMP999** [comment]
> So you want to say your PR is ready for review?

Yes

File: Telegram/SourceFiles/core/sandbox.cpp [hunk 1]
```diff
@@ -182,6 +182,9 @@ int Sandbox::start() {
 		return 0;
 	}
 	_started = true;
+
+	style::SetRightToLeft(TRUE);
```
Comment (ilya-fedin) [inline Telegram/SourceFiles/core/sandbox.cpp:186]: What's that?

Comment (ilya-fedin) [reply Telegram/SourceFiles/core/sandbox.cpp:186]: I don't think you can enable RTL layout for every user...

Comment (AMP999) [reply Telegram/SourceFiles/core/sandbox.cpp:186]: I have set that true to see how my changes appear in run time. 
It should'nt hard coded here. This function should bind to languages

Comment (ilya-fedin) [reply Telegram/SourceFiles/core/sandbox.cpp:186]: Why there's code that shouldn't be here if it's ready for review? :thinking:

Comment (AMP999) [reply Telegram/SourceFiles/core/sandbox.cpp:186]: You're right. I'll fix it and PR it again

**Pull request closed without merging**